packages/app/tests/e2e/features/redirection/redirectionSet1.feature

```diff
@@ -21,13 +21,13 @@ Feature: Redirection
   @id231
   Scenario Outline: Verify redirection for "<Icon>" social network icon  on Header
     When I click by element with partial href "<Icon>"
-    Then New page have "<url>" address
+    Then New page address matches the "<regexp>"
 
     Examples:
-      | Icon    | url                                |
+      | Icon    | regexp                                |
       # discord renamed to "join"
-      | join    | https://join.zksync.dev/           |
-      | twitter | https://x.com/zksync               |
+      | join    | ^https://join.zksync.dev/$           |
+      | x.com | ^https://x.com/zksync(\\?.*)?$               |
```

Comment:
X redirects to `https://x.com/zksync` with a query string, since it's not clear what that query string means and if it will change or not I decided to check based on the RegExp.